packages/assets-controllers/src/MultichainAssetsRatesController/MultichainAssetsRatesController.ts

```diff
@@ -280,9 +297,6 @@ export class MultichainAssetsRatesController extends StaticIntervalPollingContro
     const releaseLock = await this.#mutex.acquire();
 
     return (async () => {
-      if (!this.isActive) {
-        return;
-      }
```

Comment:
Just double checking if this was removed intentionally.

Reply:
Just pushed a fix for that, it was not intentional 🙏